.github/workflows/ci-build.yml

```diff
@@ -40,8 +40,16 @@ jobs:
         key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
         restore-keys: ${{ runner.os }}-composer-
 
+    - name: Remove dependencies not used in this job for PHP 8 compatibility
```

Comment:
FTR, none of those dependencies are used anyway in this jobs matrix so the argument could be made we always want to remove them, speeds up composer (no resolving, no installing, no downloading, no caching, etc.).

But otherwise it could be deemed as temporarily solution until the dependency issues can be resolved.

Reply:
I'd keep it as a temporary solution as it would introduce more complexity into CI. As for speed, it's micro optimalization and therefore not significant IMO.

Reply:
Roger that!